.taskcluster.yml

```diff
@@ -213,10 +229,14 @@ tasks:
                                     then: >
                                         PIP_IGNORE_INSTALLED=0 pip install --user /builds/worker/checkouts/taskgraph &&
                                         cd /builds/worker/checkouts/src &&
+                                        rm -rf taskcluster &&
+                                        ln -s /builds/worker/checkouts/template/taskcluster taskcluster &&
```

Comment:
You should not need to do this dance moving around the `taskcluster` directory. You can pass `--root path/to/taskcluster/ci` to `taskgraph decision` to read from a different location (this is how comm-central handles things). 

Alternatively, you could run taskgraph from the template repository, with the xpi repository in a known place. (This is closer to what ci-admin/ci-config does, but ci-config isn't even checked out).

Reply:
Looks like this breaks any automation on xpi-template itself, but perhaps we want to turn that off anyway?

Reply:
Looks like it breaks for test-xpi-public as well. I'm guessing --root also keeps us from looking in test-xpi-public for the xpi source, and instead looks in xpi-template ?

Reply:
> Alternatively, you could run taskgraph from the template repository, with the xpi repository in a known place. (This is closer to what ci-admin/ci-config does, but ci-config isn't even checked out).

I think I was avoiding this, because a) we need to translate paths, and b) xpi-template is level 3; the xpi source repo will be level 1. We could make xpi-template level 1. Is going back to the taskcluster/ dance a blocker?

Reply:
busted test-xpi-public: https://github.com/escapewindow/test-xpi-public/pull/6

Reply:
https://github.com/mozilla-extensions/xpi-template/issues/7

Reply:
`--root` seems to break everything. I'm going to punt on that.